.gitignore

```diff
@@ -1,6 +1,7 @@
 .DS_Store
 node_modules
 /dist
+coverage
```

Comment:
Is coverage part of this work?

Reply:
Shhhh (no, but that was easier to do now due to my local status)